docs/UserGuide/Concept/Data Model and Terminology.md

```diff
@@ -78,7 +80,18 @@ It is worth noting that in the path, root is a reserved character, which is only
 
 Single quotes are not allowed in the path. If you want to use special characters such as "." in LayerName, use double quotes. For example, `root.sg."d.1"."s.1"`. 
 
-> Note: the LayerName of storage group can only be characters, numbers and underscores.
+The characters supported in LayerName without double quotes are as below:
+
+* Chinese characters '\u2E80' to '\u9FFF'
+* '+', '&', '%', '$', '#', '@', '/', '_', '-', ':'
+* 'A' to 'Z', 'a' to 'z', '0' to '9'
+* '[', ']' (eg. 's[1', 's[1]', s[ab]')
+
+'-' and ':' cannot be the first character.
```

Comment:
hi, 
root.sg3.+ not supported.
but root.sg3.++ is okay..

Reply:
it is strange .... should be consistent.